src/css/fonts.css

```diff
@@ -0,0 +1,41 @@
+@charset "UTF-8";
+
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i,900,900i");
+@import url("https://fonts.googleapis.com/css?family=Source+Serif+Pro:400,600,700");
+@import url("https://fonts.googleapis.com/css?family=Roboto+Slab:300,400,700");
+@import url("https://fonts.googleapis.com/css?family=Kalam:300,400,700");
```

Comment:
Are we still supporting Kalam and Noto Sans for v7?

Reply:
I'm not sure, I would like to keep a 1:1 with v6 where possible but if we are going to remove them, we should make a note as deprecated in v6.

Reply:
Who gets to make that call? @kerri-augenstein perhaps?

Reply:
Yeah @kerri-augenstein and Dana at UComm both agreed to it in a previous discussion, but let's confirm with them again before making any decisions. Please feel free to leave them in for this PR.

Reply:
> Kerri Augenstein  @Shea we haven't needed script at all, so no on Kalam, and I have no recollection of the Noto Sans and where that came from. I'd say no on that as well.